apps/dev-api/domains/miniapp/schema/ImportB2CAppService.js

```diff
@@ -235,6 +236,69 @@ async function importAppBuilds ({ args, context }) {
     }
 }
 
+async function _importAppAccessRightFromEnvironment ({ condoAppId, context, environment, args }) {
+    const { data: { dv, sender, to: { app: { id: appId } } } } = args
+
+    const exportField = `${environment}ExportId`
+
+    const serverClient = environment === PROD_ENVIRONMENT
+        ? productionClient
+        : developmentClient
+
+    const accessRights = await serverClient.getModels({
+        modelGql: CondoB2CAppAccessRightGQL,
+        where: { app: { id: condoAppId } },
+        first: 1,
+    })
+
+    if (accessRights.length) {
+        const condoAccessRight = accessRights[0]
+
+        const accessRight = await B2CAppAccessRight.create(context, {
+            dv,
+            sender,
+            condoUserId: condoAccessRight.user.id,
+            app: { connect: { id: appId } },
+            environment,
+            [exportField]: condoAccessRight.id,
+        })
+
+        await serverClient.updateModel({
+            modelGql: CondoB2CAppAccessRightGQL,
+            id: condoAccessRight.id,
+            updateInput: {
+                dv,
+                sender,
+                importId: accessRight.id,
+                importRemoteSystem: REMOTE_SYSTEM,
+            },
+        })
+    }
+}
+
+async function importAppAccessRight ({ args, context }) {
+    const developmentAppId = get(args, ['data', 'from', 'developmentApp', 'id'])
+    const productionAppId = get(args, ['data', 'from', 'productionApp', 'id'])
+
+    if (developmentAppId) {
+        await _importAppAccessRightFromEnvironment({
+            condoAppId: developmentAppId,
+            environment: DEV_ENVIRONMENT,
+            args,
+            context,
+        })
+    }
+
+    if (productionAppId) {
+        await _importAppAccessRightFromEnvironment({
+            condoAppId: productionAppId,
+            environment: PROD_ENVIRONMENT,
+            args,
+            context,
+        })
+    }
```

Comment:
Is there an option to a third environment? If no, you can easily do that stuff inside one block of the code. 
Or you want to "import environment" only for dev & production? And, for example, for test proposes there is no need to call this function here?

Reply:
You can have different users for production and development environment. So, importing app = import all its information from single of both environments including `B2CAppAccessRight`. It can be different (at least user ids in condo.d and v1 are not the same), so we need to import both.